templates/push.html

```diff
@@ -5,4 +5,4 @@
 {%- endif -%}
 \[{{repo_name}}] \[{{repo}}/{{branch}}] {{user}} pushed {{commits}} {{commit}}:
 {% for c in commit_messages %}
-  - [{{c.hash}}]({{c.url}}) {{c.msg}}{% endfor %}
+  - {{c.url}} {{c.msg}}{% endfor %}
```

Comment:
Now with your previous comment, I understand why the long URL didn’t bother anyone, it’s because the hash is supposed to be a link, and the URL is hidden.

I use XMPP, and it doesn’t render markdown at all, that’s why I made this change (the URL is displayed, and it’s clickable).

So yeah, maybe this change is not ideal (and thus the other one you commented on).

I’ll take a look at errbotio/errbot#879 to see if we can have a better solution.